app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```
chạy rubocop và fix đã nhé, mấy cái này rubocop bắt rồi, ko tớilượt chị mô :D